lib/SILGen/SILGen.cpp

```diff
@@ -657,10 +657,13 @@ void SILGenModule::preEmitFunction(SILDeclRef constant,
              F->getLoweredType().print(llvm::dbgs());
              llvm::dbgs() << '\n';
              if (astNode) {
-               if (auto *decl = astNode.dyn_cast<ValueDecl *>())
+               if (auto *decl = astNode.dyn_cast<ValueDecl *>()) {
                  decl->dump(llvm::dbgs());
-               else
+               }
+               else {
```

Comment:
Nitpick: We put `} else {` all on one line.

Reply:
You're right, thanks :)
Should I make the changes you mentioned and create a new pull request, or would you rather fix them on your end?

Reply:
I think it's easiest for you to make them and update this pull request (by pushing to the same branch).